docs/config.md

```diff
@@ -34,8 +35,8 @@ rdap:
 
         - name: arin
           servers:
-              - https://rdap.arin.net/registry
-              - http://rdap.arin.net/registry
+              - https://rdap.arin.net/registry/
```

Comment:
If the servers strings need the terminating slash, I would suggest either documenting this requirement or modifying the (I'm assuming) joining code to account for it.